<launch>

<!-- <arg name="use_sim_time" default="false"/> -->
<node name="pure_p" pkg="pure_p" type="pure_p" respawn="false" output="screen">
  
    <param name = "wheelbase" value = "0.25"/>
    <param name = "lookahead_distance" value = "0.5"/>
    <param name = "~/linear_velocity" value = "0.3"/>

    <!-- <remap from="/pathsmooth" to="/planning/planning/execute_path" /> -->
    <remap from="/pathsmooth" to="/planning/planning/execute_path_op" />
     <remap from="/odom" to="/aft_mapped_to_init" />
    <param name = "max_rotational_velocity" value = "0.5"/>
    <param name = "position_tolerance" value = "0.1"/>
    <param name = "steering_angle_velocity" value = "100.0"/>
    <param name = "acceleration" value = "100.0"/>
    <param name = "jerk" value = "100.0"/>
    <param name = "steering_angle_limit" value = "1.57"/>
    <!-- <param name = "map_frame_id" value = "map"/> -->
    <!-- <param name = "robot_frame_id" value = "base_link"/> -->
    <param name = "map_frame_id" value = "map"/>
    <param name = "robot_frame_id" value = "base"/>
</node>
  
</launch>
